pandas/core/internals/ops.py

```diff
@@ -127,3 +134,17 @@ def blockwise_all(left: "BlockManager", right: "BlockManager", op) -> bool:
         if not res:
             return False
     return True
+
+
+def array_equals(left: ArrayLike, right: ArrayLike) -> bool:
```

Comment:
since the inputs are `ArrayLike` and not block related, would it make sense to co-locate this with `array_equivalent` in `core\dtypes\missing.py`

Reply:
good idea